src/context/TransactionsProvider/TransactionsProvider.tsx

```diff
@@ -160,6 +160,7 @@ export const TransactionsProvider: React.FC<TransactionsProviderProps> = ({ chil
 
   const maybeRefetchVotingPower = useCallback(
     ({ status }: Transaction, chainId: ChainId) => {
+      if (!isConnected) return
```

Comment:
Don't fetch vp without a wallet, which would result in perma-loading states in e.g LP